scripts/exgdas_atmos_verfozn.sh

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
```

Comment:
Same comment as above. Can't use preamble but thankfully it looks like this script hasn't changed much between v16 and v17 so you can likely keep most of the v17 version:
```
[kfriedma@hercules-login-2 scripts]$ pwd
/work/noaa/global/kfriedma/git_hercules/dev-gfs.v16.GSIdev/scripts
[kfriedma@hercules-login-2 scripts]$ diff ../../release-gfs.v16.3.21/scripts/exgdas_atmos_verfozn.sh exgdas_atmos_verfozn.sh
1c1
< #/bin/sh
---
> #! /usr/bin/env bash
3c3
< set -ax
---
> source "$HOMEgfs/ush/preamble.sh"
12,13d11
< export scr=exgdas_vrfyozn.sh
<
73c71
<    count=`ls diag* | grep ".nc4" | wc -l`
---
>    count=$(ls diag* | grep ".nc4" | wc -l)
76,77c74,75
<       for filenc4 in `ls diag*nc4.gz`; do
<          file=`echo $filenc4 | cut -d'.' -f1-2`.gz
---
>       for filenc4 in $(ls diag*nc4.gz); do
>          file=$(echo $filenc4 | cut -d'.' -f1-2).gz
92,98d89
<
< if [[ "$VERBOSE" = "YES" ]]; then
<    echo "end exgdas_vrfyozn.sh, exit value = ${err}"
< fi
<
<
< set +x
```